engine/api/scheduler/dao.go

```diff
@@ -233,7 +233,7 @@ func LoadPastExecutions(db gorp.SqlExecutor, id int64) ([]sdk.PipelineSchedulerE
 //LoadPendingExecutions loads all pipeline execution
 func LoadPendingExecutions(db gorp.SqlExecutor) ([]sdk.PipelineSchedulerExecution, error) {
 	as := []PipelineSchedulerExecution{}
-	if _, err := db.Select(&as, "select * from pipeline_scheduler_execution where executed = 'false' and execution_planned_date <=  $1", time.Now()); err != nil {
+	if _, err := db.Select(&as, "select * from pipeline_scheduler_execution where executed = 'false' and execution_planned_date <= now()"); err != nil {
```

Comment:
if you compare execution_planned_date with the database time, you should insert the database time